### ISSUE TITLE
Missing exceptional items in the registry

### ISSUE THREAD

**Issue by ygonno** [ISDCF/registries#47]
I notice there are a couple of missing exceptional items in the registry while those are in the Google Sheets. I wonder if we want to put them in the JSON version with notes. So far I found the followings are missing:

### Facilities:
CODE | DESCRIPTION
NUL | Unspecified

### Studios:
CODE | DESCRIPTION
NULL | Studio Not Specified
INDI | Independent Film Studio not specified

### Language Codes:
MOS | NO SOUND AT ALL (Silent)
XX | NO SPOKEN LANGUAGE | No Spoken language (but not silent) or No Subtitles

**Comment by ygonno** [comment]
I found the below is done in the latest version. Ones for Language Codes still remain missing.

> ### Facilities:
> CODE | DESCRIPTION
> NUL | Unspecified
> 
> ### Studios:
> CODE | DESCRIPTION
> NULL | Studio Not Specified
> INDI | Independent Film Studio not specified

**Comment by ygonno** [comment]
As these are exceptional items, to draw special attention, should they be on top of the list as in the Google Sheets versions?

> I found the below is done in the latest version. Ones for Language Codes still remain missing.
> 
> > ### Facilities:
> > CODE | DESCRIPTION
> > NUL | Unspecified
> > ### Studios:
> > CODE | DESCRIPTION
> > NULL | Studio Not Specified
> > INDI | Independent Film Studio not specified

**Comment by SteveLLamb** [comment]
Null as well as the MOS and XX are not there internationally. The absence of the metadata fields mean null, not specified. As far as the values of the CTT, we can add special notes to the DCNC site to state these cases, but I don’t think they should be in the registry. @palemieux and @radford-for-smpte would you agree? 

As far as indies, I do agree that should be in there.

**Comment by japierce1028** [comment]
I think we need NULL, MOS and XX in the registry. Since these values may appear in the human readable name, I think they should be able to look them up with all the rest, not a side note.

**Comment by SteveLLamb** [comment]
In that case, there will need to be a note for each value that this represents the values for the CTT only and that the metadata fields should be not present to denote null, xx, or mos (not present).

**Comment by SteveLLamb** [comment]
FYI - for studio and facility I am adding the following comment:

"comments": [
      "This code is only use in the CTT. If using 429-16 metadata, the absence of the <Facility> element shall denote the facility is 'Unspecified'"
    ]

"comments": [
      "This code is only use in the CTT. If using 429-16 metadata, the absence of the <Distributor> element shall denote the studio is 'Unspecified'"
    ]

However, the issue with Languages is that everything is keyed off of the RFC tags. As mentioned i many past ISDCF meetings, MOS is actually a language tag:

Type: language
Subtag: mos
Description: Mossi
Added: 2005-10-16

XX is not an available RFC tag. I'm not sure of the best path forward with those two items in language (NULL is not used there).

**Comment by SteveLLamb** [comment]
After a lengthy chat with Pierre, we figured out that MOS actually shouldn't be in the language fields, that it is actually supposed to be in the sound config fields (and verified it is). We'll work on hardcoding in the XX for the language with a Null value or similar, with adding in comments that are like for the Null in Facility and Studio.

**Comment by ygonno** [comment]
I understand MOS should be in the Audio Type field, not in the Language field.

How about the data locations of NULL and INDI in JSON?  Shouldn't those be located at the top of the list? Or should I put a side note to explain such exceptional cases?

**Comment by SteveLLamb** [comment]
While doing other research on language tags, I came across this in IANA:

Type: language
Subtag: zxx
Description: No linguistic content
Description: Not applicable
Added: 2006-03-08
Scope: special
%%

I suggest using this subtag to denote our previously used "XX", and keep the "XX" for the DCNC only. The Description is suitable for our purposes mapping to CLDR, and we can make a note that is to be used for "no language", and that metadata fields should just not be used (they are optional etc) such as:

"comments": [
"This code is only use in the CTT. If using 429-16 or 429-7 metadata, the absence of the <Language> element shall denote the language is 'None'"
]

I still don't have a good solution for MOS, but not sure it is really needed, other than in an informational note outside the tables. 

Thoughts?
